tests/target/issue-447.rs

```diff
@@ -0,0 +1,7 @@
+fn main() {
+    if cond {
+    }
+    // This shouldn't be dropped
+    else {
+    }
+}
```

Comment:
Could you add a few more tests please? I'd like to see a `/* */` comment where `} else {` and the comment are all on the same line. Are there other ways a comment could appear here?